plugins/src/ios/InitializePluginManager.m

```diff
@@ -0,0 +1,27 @@
+//
+//  InitializePluginManager.m
+//  Example_App
+//
+//  Created by Barak Ben Hur on 03/08/2022.
+//
+
+#import "InitializePluginManager.h"
+#import "Example_App-Swift.h"
+
+@implementation InitializePluginManager
+
++ (void)load {
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{
+        [NSNotificationCenter.defaultCenter addObserver: self
+                                               selector: @selector(didFinishLaunching:)
+                                                   name: UIApplicationDidFinishLaunchingNotification
+                                                 object: nil];
+    });
+}
+
++ (void) didFinishLaunching: (NSNotification*) n {
+    [[Optimove_Cordova alloc] pluginInitialize];
```

Comment:
We discussed not creating a second instance of this plugin, has been refactored in DDL branch so will be reconsidered as part of that scope.

Reply:
After further discussion, we will refactor this just now on this branch, to:

- Call a static method on Optimove_Cordova that will perform SDK initialisation
- Capture a static reference to the plugin instance from the `pluginInitialize` override in the plugin class

Reply:
fixed

Reply:
OK I think we lost the part that calls the initialise of the SDK from applicationDidFinishLaunching?

Reply:
add them back